src/classBased/components/InputTodo.js

```diff
@@ -26,6 +26,7 @@ class InputTodo extends Component {
         title: '',
       });
     } else {
+      // eslint-disable-next-line
       alert('Please write item');
     }
   };
```

Comment:
- [ ] You have a folder with classbased components files. It would be nice if you remove this folder if these files are unnecessary. If you have any necessary file with class-based component in this folder then convert that into functional component :blush: